kymatio/scattering2d/backend/numpy_backend.py

```diff
@@ -34,8 +34,10 @@ def __call__(self, x):
         if self.pre_pad:
             return x
         else:
-            np_pad = ((self.pad_size[0], self.pad_size[1]), (self.pad_size[2], self.pad_size[3]))
-            output = np.pad(x, ((0,0), np_pad[0], np_pad[1]), mode='reflect')
+            paddings = ((0, 0),)
```

Comment:
outer parentheses not really necessary

Reply:
It appears they are ( i get errors if I do not have outer parens)

Reply:
Did you try: `paddings = (0, 0),`

The right-most comma is required.

Reply:
right, thanks for the reminder.

Reply:
I'm fine either way